scapy/utils.py

```diff
@@ -1534,16 +1534,20 @@ def _write_packet(self, packet, sec=None, usec=None, caplen=None,
 
 
 @conf.commands.register
-def import_hexcap():
+def import_hexcap(s=None):
```

Comment:
Could you use a meaningful variable name such as `input_string` and add a parameter documentation as done here https://github.com/secdev/scapy/blob/master/scapy/utils.py#L251?

Reply:
Done. I hope I did it correctly.